nameable.rb

```diff
@@ -0,0 +1,28 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
+
+class BaseDecorator < Nameable
```

Comment:
- [ ] You'll need a getter for **nameable**, that's
```ruby
class BaseDecorator < Nameable
    attr_accessor :nameable
     
      ...


  
```
Kindly fix this issue :pray: